common/protob/messages-tron.proto

```
@@ -143,6 +144,11 @@ message TronSignedTx {
     optional bytes serialized_tx = 2; // Serialized transaction
 }
 
+enum TronMessageType {
+    V1 = 1 [deprecated = true];
```

Buf (1.55.1)

First enum value "V1" should have a numeric value of 0

(ENUM_FIRST_VALUE_ZERO)